src/pages/Home.js

```diff
@@ -17,11 +17,9 @@ const Home = ({content, handleNext}) => {
                         </p>
                     ))}
                 </div>
-
-                <div className="button" href="/">
-                    <a href="/question">Start </a>
-                </div>
-
+                <button className="button button--red" onClick={() => handleNext()}>
```

Comment:
classes had ik aangepast zodat we de class --red niet meer nodig hadden, of is dat weer veranderd ?